modules/account-lib/test/unit/coin/sol/transaction.ts

```diff
@@ -542,5 +545,138 @@ describe('Sol Transaction', () => {
         durableNonce: undefined,
       });
     });
+
+    it('should activate builder ', async function () {
+      const tx = await factory
+        .getStakingActivateBuilder()
+        .stakingAddress(stakeAccount.pub)
+        .sender(wallet.pub)
+        .nonce(blockHash)
+        .amount(amount)
+        .validator(validator.pub)
+        .fee({ amount: 5000 })
+        .build();
+
+      const explainedTransaction = tx.explainTransaction();
+      explainedTransaction.should.deepEqual({
+        displayOrder: [
+          'id',
+          'type',
+          'blockhash',
+          'durableNonce',
+          'outputAmount',
+          'changeAmount',
+          'outputs',
+          'changeOutputs',
+          'fee',
+          'memo',
+        ],
+        id: 'UNAVAILABLE',
+        type: 'StakingActivate',
+        changeOutputs: [],
+        changeAmount: '0',
+        outputAmount: '10000',
+        outputs: [
+          {
+            address: '7dRuGFbU2y2kijP6o1LYNzVyz4yf13MooqoionCzv5Za',
+            amount: '10000',
+          },
+        ],
+        fee: {
+          fee: '10000',
+          feeRate: 5000,
+        },
```

Comment:
@alebusse do you think this could be related to the bug we're seeing with transfer amounts?
our `explainTransaction` impl doesn't add `fee` to `outputAmount`.